Sources/Server/Response.swift

```diff
@@ -3,55 +3,80 @@ import Foundation
 public class Response {
 
     let parser: Parser
-    var paths: Array<String>
+    var paths: [String: (Response) -> (Request) -> (String)] = ["/": handleRoot,"/hello": handleHello,"/coffee": handleCoffee, "/tea": handle200]
     let status200: String
     let status404: String
     let status404Body: String
 
     public init(parser: Parser){
         self.parser = parser
-        self.paths = ["/","/hello"]
         self.status200 = "200 OK"
```

Comment:
It might be worth thinking about making these a struct/enum referenced like `Response.statuses.200` (just a thought)